app/src/main/java/com/jasonette/seed/Component/JasonImageComponent.java

```diff
@@ -90,29 +97,29 @@ public static Object resolve_url(JSONObject component, Context context){
     private static void gif(JSONObject component, View view, Context context){
         Object new_url = JasonImageComponent.resolve_url(component, context);
         Glide
-            .with(context)
-            .load(new_url)
-            .asGif()
-            .diskCacheStrategy(DiskCacheStrategy.SOURCE)
-            .into((ImageView)view);
+                .with(context)
```

Comment:
Hrm...there's a white space change here.  I wonder if it would make sense to standardise on build settings for this repo?

Not sure how to do it via the web, though.  Can we host the settings file on github?

Reply:
I don't understand what is your proposal. What do you mean with "settings files on github"?

Reply:
I was thinking of the Settings Repository.  Preferences -> Tools -> Settings Repository.  We use it at work, and it stores the formatting choices in a repo, and retrieves them when you open the project.

That said, this article proposes sharing the settings in the .idea folder, which I find interesting:
https://tips.seebrock3r.me/share-the-settings-with-the-whole-team-android-studio-protip-6-3cce16eb2ea4

That way there's not a surprising network access...and the formattingn options are common amongst anyone who checks out the repo.